beta/examples/tensorflow/classification/configs/sparsity_quantization/resnet50_imagenet_magnitude_sparsity_int8.json

```diff
@@ -35,12 +35,11 @@
                     0.4,
                     0.5
                 ],
-                "sparsity_freeze_epoch": 30
+                "sparsity_freeze_epoch": 29
```

Comment:
The change is correct but difficult to understand. Let's update the configuration file and leave 30.

Reply:
Done